scripts/asm-installer/cloudbuild.yaml

```diff
@@ -21,13 +21,16 @@ steps:
   - '${_SCRIPT_NAME}'
   waitFor:
   - 'publish-tester-image'
-- name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
-  dir: 'scripts/asm-installer'
-  id: 'check-posh-compatibility'
-  entrypoint: 'posh'
-  args: ['-n', '${_SCRIPT_NAME}']
-  waitFor:
-  - 'publish-tester-image'
+
+    # Disabled for now, see #296. We have more users with LTS Linux releases
+    # than non-Bourne shells. Please open an issue if you're affected.
+    #- name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
+    #  dir: 'scripts/asm-installer'
+    #  id: 'check-posh-compatibility'
+    #  entrypoint: 'posh'
+    #  args: ['-n', '${_SCRIPT_NAME}']
+    #  waitFor:
```

Comment:
How is the posh compatibility check related to the curl command update?

Reply:
The <(command) construction is bash specific so it posh fails on it. I still want to have this at some point, but until we come up with a POSIX compatible way or we have no more users with curl versions < 7.55 then I think we have to give it up.

Reply:
got it, thanks!